gw_eccentricity/load_data.py

```diff
@@ -1366,13 +1366,9 @@ def get_num_orbits_duration_from_horizon_data(horizon_filepath, num_orbits):
     xB_data = horizons_data["AhB.dir"]["CoordCenterInertial.dat"]
     time = xA_data[:, 0]
     separion_vec = xA_data[:, 1:] - xB_data[:, 1:]
-    # We will assume the system to be non-precessing so that the motion
-    # is restricted in the x-y plane and the z-coordinate is effectively zero.
-    # Check if it is true, i.e., z-coordinate remains very small
-    if any(np.abs(separion_vec[:, 2]) > 1e-5):
-        raise Exception(
-            "The system seems to be precessing. Cannot get the orbital "
-            "phase by assuming the motion is restricted in the x-y plane.")
+    # We will use the x-y coordinates, i.e., the motion projected
+    # onto the x-y plane to compute the phase and then the number of
+    # orbits using that phase.
```

Comment:
Call it phase_orb_projected so that it is not accidentally used for something else.

Reply:
Maybe add a comment saying that this assumption breaks down when the orbital plane itself contains the z-axis, which can happen if there is extreme precession, or in the case of EMRIs.

Reply:
Done